bin/static-review.php

```diff
@@ -12,9 +12,17 @@
  * @see http://github.com/sjparkinson/static-review/blob/master/LICENSE
  */
 
-$included = include file_exists(__DIR__ . '/../autoload.php')
-    ? __DIR__ . '/../autoload.php'
-    : __DIR__ . '/../../../autoload.php';
+$currentDir = dirname(__DIR__);
+$included = false;
+while ($currentDir !== '/') {
+    if (file_exists($currentDir . '/composer.json')) {
+        $vendorDir = trim(shell_exec('composer config vendor-dir'));
```

Comment:
@shadowhand is this really unconventional? It does solve the problem of exotic composer configurations